app/views/admin/statistics_announcements/_search_results.html.erb

```diff
@@ -0,0 +1,59 @@
+<p class="govuk-heading-s govuk-!-margin-bottom-3"><%= pluralize(number_with_delimiter(paginator.total_count), "document") %></p>
+
+<% if paginator.blank? %>
+  <p class="govuk-body app-view-edition-search-results__no_documents">No <%= filter.options[:dates] %> statistics announcements found</p>
+<% else %>
+  <div class="govuk-table--with-actions">
+    <%= render "govuk_publishing_components/components/table", {
+      head: [
+        {
+          text: "Title"
+        },
+        {
+          text: "Organisation"
+        },
+        {
+          text: "Release date"
+        },
+        {
+          text: "Announcement"
+        },
+        {
+          text: tag.span("View", class: "govuk-visually-hidden")
+        }
+      ],
+      rows:
+        paginator.map do |statistics_announcement|
+          [
+             {
+               text: tag.strong(statistics_announcement.title)
+             },
+             {
+               text: statistics_announcement.organisations.map(&:name).to_sentence
+             },
+             {
+               text: sanitize("#{statistics_announcement.release_date.to_date.strftime("%e&nbsp;%B&nbsp;%Y").html_safe}
+                              <br>
+                              #{statistics_announcement.release_date.strftime("%l:%M%P")}
+                              (#{statistics_announcement.confirmed? ? 'confirmed' : 'provisional'})"
```

Comment:
So with this we could do 
```
text: tag.p(statistics_announcement.release_date.to_date.strftime(sanitize("%e&nbsp;%B&nbsp;%Y")), class: "govuk-!-margin-0") +
tag.p(statistics_announcement.release_date.strftime("%l:%M%P"), class: "govuk-!-margin-0") +
 "(#{statistics_announcement.confirmed? ? 'confirmed' : 'provisional'})"
```

Using `strftime("%e&nbsp;%B&nbsp;%Y")` to stop a line break feels a bit wrong though.

I understand without it if you use  `strftime("%e %B %Y")` you get 

<img width="121" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/1e0dc03c-4b79-46cf-862d-c0ea8fce77ce">

as it's not a continuous string so you get line breaks which looks bad.

 I think the correct thing to do is set a width on the column in the stylesheet, or give the p a class which you can style to not have line break. The column width is probably easier and we have prior art for though.